### ISSUE TITLE
[Bug] Precision Tracer Cache Miss

### ISSUE THREAD

**Issue by Prayer3th** [sgl-project/sglang-jax#130]
### Checklist

- [x] 1. I have searched related issues but cannot get the expected help.
- [x] 2. The bug has not been fixed in the latest version.
- [x] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.
- [x] 4. If the issue you raised is not a bug but a question, please raise a discussion at https://github.com/sgl-project/sgl-jax/discussions/new/choose Otherwise, it will be closed.
- [x] 5. Please use English, otherwise it will be closed.

### Describe the bug

Precision Tracer Cache Miss on forward

### Reproduction

git pull origin fix/tracer3 --rebase && JAX_COMPILATION_CACHE_DIR=/tmp/jit_cache  python3 -u -m sgl_jax.launch_server --model-path Qwen/Qwen3-8B --trust-remote-code  --dist-init-addr=0.0.0.0:10011 --nnodes=1  --tp-size=4 --device=tpu --random-seed=3 --node-rank=0 --mem-fraction-static=0.8 --chunked-prefill-size=8192 --download-dir=/tmp --dtype=bfloat16 --max-running-requests 16 --max-total-tokens 257536 --skip-server-warmup --attention-backend=fa  --page-size=64 --enable-precision-tracer --disable-radix-cache --precompile-token-paddings 8192 --precompile-bs-paddings 16

### Environment

google v6e * 4
